src/UI/Art/Animation/BaseUnitActionAnimation.java

```diff
@@ -4,9 +4,7 @@
 import java.util.ArrayList;
 
 import Engine.XYCoord;
-import UI.Art.SpriteArtist.SpriteLibrary;
-import UI.Art.SpriteArtist.SpriteMapView;
-import UI.Art.SpriteArtist.UnitSpriteSet;
+import UI.Art.SpriteArtist.*;
```

Comment:
I'm not sure we approve of this kind of thing around here.

Reply:
Why not? It means less churn on the imports, and no annoying warnings after you remove the last reference to something... *like that one we have in Unit ;-;*.

Reply:
It's generally considered bad practice to import with wildcards. It clutters the namespace with a bunch of stuff you aren't using, makes name collisions more likely, and makes it harder to see what you actually use from the package. Plus I just prefer to be explicit about things.

Reply:
The clutter is mostly irrelevant... especially in namespaces that *we* define.
Collisions are mostly not a thing we need to worry about, especially if you "be explicit about things" and specify the enclosing scope next to the invocation.

Also, I literally never look at the imports unless the IDE tells me something's wrong with them.
The signal to noise ratio is awful.
Fiddling with that less often is a win in my book.

Reply:
Blah fine. Hopefully we don't find reason to regret it.